Framework/PythonInterface/test/python/plugins/algorithms/ReflectometryReductionOneLiveDataTest.py

```diff
@@ -203,13 +228,22 @@ def test_algorithm_fails_for_invalid_block_names(self):
                           Instrument='INTER',
                           GetLiveValueAlgorithm='GetFakeLiveInstrumentValueInvalidNames')
 
+    def test_algorithm_fails_if_theta_is_zero(self):
```

Comment:
I think this should say something like `test_algorithm_passes_if_theta_is_zero` or similar.